rules/lowercase_description.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const ruleMsg = 'it() description should begin with lowercase';
+
+const isItOrTestorDescribeFunction = node => {
+  return (
+    node.type === 'CallExpression' &&
+    node.callee &&
+    (node.callee.name === 'it' ||
+      node.callee.name === 'test' ||
+      node.callee.name === 'describe')
+  );
+};
+
+const isItDescription = node => {
+  const foo =
```

Comment:
Minor suggestion: return this conditional statement instead of assigning to `foo`, since the variable doesn't explain what's going on in this function.

Reply:
Ops, forgot to remove this when I poking around. Nice catch.